cowprotocol/cowswap
# fix(limit-orders): fix tooltips displaying in settings

# Summary

Before the fix:
![image](https://github.com/cowprotocol/cowswap/assets/7122625/070240bd-419f-4d18-a85b-7adfd12837dd)

after the fix:
<img width="611" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/efff2679-b699-4367-99b2-e18de7e81bfc">


  # To Test

1. Open settings in Limit orders
2. Hover on some `?` icon
- [ ] AR: tooltip is displayed in the left top window corner
- [ ] ER: tooltip is displayed near `?` icon

  # Background

The bug was introduced in the recent NX updates.

<img width="450" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/ba82ab1f-63d5-4c99-9cf7-a67146347a34">
